app/views/play/level/tome/TomeView.coffee

```diff
@@ -118,7 +118,10 @@ module.exports = class TomeView extends CocoView
     else
       classroomCodeFormatDefault = @options.classroomAceConfig?.codeFormatDefault or 'text-code'
       classroomCodeFormats = @options.classroomAceConfig?.codeFormats or ['blocks-icons', 'blocks-text', 'blocks-and-code', 'text-code']
-      desiredCodeFormat = me.get('aceConfig')?.codeFormat or classroomCodeFormatDefault
+      desiredCodeFormat = me.get('aceConfig')?.codeFormat
+      if @options.level.get('product') == 'codecombat-junior'
+        desiredCodeFormat ?= 'blocks-icons'
+      desiredCodeFormat ?= classroomCodeFormatDefault
       newCodeFormat = if desiredCodeFormat in classroomCodeFormats then desiredCodeFormat else classroomCodeFormatDefault
```

Comment:
I'm not sure what is the objective here, but this line will reassign it back to `text-code` if `classroomCodeFormats` doesn't contain `block-icons`

Reply:
ya it is. if the teacher want classroom don't use block they can still disable that so that junior will still use text-code